lib/converter.js

```diff
@@ -24,7 +25,104 @@ function registerTemplateConverter (processor, templates) {
   processor.ConverterFactory.register(new TemplateConverter(), ['html5'])
 }
 
-async function convert (processor, inputFile, options, timings) {
+function getOutline (node, depth) {
```

Comment:
Could you please extract the below functions to a dedicated file `outline.js` that exports the `addOutline` function?

Reply:
sure. fixed

Reply:
:+1: